### ISSUE TITLE
songwhip bot appears to be down '

### ISSUE THREAD

**Issue by nobuhojimichaan** [vleeuwenmenno/songwhip-bot#18]
hi i'm not really sure where else to report this but plz let me know if there's a better place to ask '   it appears the songwhip bot is offline ! '

**Comment by vleeuwenmenno** [comment]
Correct, it'll be back soon. I'm moving and it was hosted at my home. I'll move the bot to a VPS so it will run more reliably.

**Comment by vleeuwenmenno** [comment]
It should be back online 👍🏻

**Comment by Podrock** [comment]
It is - thank you !
